client/src/pages/Cart.jsx

```diff
@@ -6,77 +6,96 @@ import { usePayment } from '../features/cart/usePayment';
 import Spinner from '../ui/Spinner';
 import { formatCurrency, formatDateTimeDetailed } from '../utils/helpers';
 import { Link } from 'react-router-dom';
+import { Context } from '../main';
+import { useContext } from 'react';
 
 export default function Cart() {
+  const { isDarkMode } = useContext(Context);
+
   const { cart, isLoading } = useCart();
   const { deleteFromCart, isDeleting } = useDeleteCartItem();
   const { clearCart, isClearing } = useClearCart();
   const { initiatePayment } = usePayment();
 
   return (
     <PageLayout>
-      <h2 className="text-3xl font-bold mt-4">Your Cart</h2>
-      {isLoading ? (
-        <Spinner />
-      ) : (
-        <div className="flex flex-col gap-4 mt-4">
-          {cart.eventIds.length === 0 ? (
-            <p className="mt-2">
-              There are no items present your cart!
-              <Link to="/events" className="text-blue-500 hover:text-blue-700">
-                {' '}
-                Add new items to your car &rarr;
-              </Link>
-            </p>
-          ) : (
-            <>
-              {cart.eventIds.map((event) => (
-                <div key={event._id} className="flex gap-4">
-                  <img
-                    src={event.cardImage}
-                    className="w-28 h-28 rounded-lg"
-                    alt=""
-                  />
-                  <div className="flex flex-col justify-between py-4">
-                    <p className="text-lg">{event.name}</p>
-                    <p className="text-primary-700 font-light">
-                      {formatCurrency(event.eventCharges)}
-                    </p>
-                    <p className="text-primary-700 font-light">
-                      {formatDateTimeDetailed(event.date)}
-                    </p>
+      <div
+        className={`w-[80%] mx-auto p-6 transition-colors duration-300 ${
+          isDarkMode ? 'bg-gray-800 text-white' : 'bg-gray-100 text-black'
```

Comment:
Hey @kumartushar0605,

This is a great start! I noticed that we're using conditional classes to check for dark mode and apply the appropriate styles. I would recommend simplifying this by leveraging Tailwind's built-in dark mode utility classes. For example, instead of conditional logic, you could use:

```suggestion
'dark:bg-gray-800 dark:text-white bg-gray-100 text-black'
```

Tailwind's dark mode utilities make it easier to manage dark mode styles without needing extra JavaScript or conditionals, improving readability and maintainability. Plus, it keeps everything consistent across components.

You can find more information on Tailwind's dark mode utilities here: [Tailwind Dark Mode Docs](https://tailwindcss.com/docs/dark-mode).

Let me know what you think!

cc: @PorePranav

Reply:
Thank you for your inputs. This is exactly what I addressed in the first PR review. If you are interested to work on this along with @kumartushar0605 you guys can collaborate. I'll make sure that both of you get the credit!

Reply:
Okay, cool. I’m interested with working to fix the issues.
Make sense, thank you.

Reply:
Do I create a fork of the repo? Or perhaps it’s based off creating branch off main?

Reply:
How about I pull @kumartushar0605 's PR to dev branch, and then you pull it from there to fix the changes

> Do I create a fork of the repo? Or perhaps it’s based off creating branch off main?

Reply:
> How about I pull @kumartushar0605 's PR to dev branch, and then you pull it from there to fix the changes
> 
> > Do I create a fork of the repo? Or perhaps it’s based off creating branch off main?

That makes sense, appreciate that. Thanks, what’s the dev branch name?

Reply:
> > How about I pull @kumartushar0605 's PR to dev branch, and then you pull it from there to fix the changes
> > > Do I create a fork of the repo? Or perhaps it’s based off creating branch off main?
> 
> That makes sense, appreciate that. Thanks, what’s the dev branch name?

I have merged this PR into dev branch of the main repository, kindly pull from there and you can then start your work. Thank you!